src/locales/de.json

```diff
@@ -274,9 +289,35 @@
       "note": "The cron user will also need sudo authority to interact with the Red Hat Insights Client. Below is an example of the addition need to the /etc/sudoers file to provide password-less sudo for an example user ocpcollector:",
       "note_title": "Note:"
     },
+    "enable_account_access": {
+      "arn_helper_text": "e.g. arn:aws:iam:589173575009:role/CostManagement",
```

Comment:
Please change the ARN to something more generic and like `arn:aws:iam:1233456789:role/CostManagement`